docs/src/apalache/antipatterns.md

```diff
@@ -29,7 +29,7 @@ min(S) ==
     IN IF e < minOther THEN e ELSE minOther 
 ```
 
-Apalache dislikes the use of the above, for several reasons. Firstly, since the operator is `RECURSIVE`, Apalache requires a predefined upper bound on unrolling, which means that the user must know, ahead of time, what the largest `|S|` is, for any set `S`, to which this operator is ever applied. 
+Apalache dislikes the use of the above, for several reasons. Firstly, since the operator is `RECURSIVE`, Apalache would not support it after 0.23.1. In earlier versions Apalache requires a predefined upper bound on unrolling, which means that the user must know, ahead of time, what the largest `|S|` is, for any set `S`, to which this operator is ever applied. 
```

Comment:
Is there any value to discussing what Apalache did in the past?

Reply:
Well, in this particular case (and one other IIRC) there's a running example in the manual page that uses recursion in some way, so it's easier to just write  that this particular example wouldn't work today, because it's recursive, but still keep the rest of the section unchanged and explain whatever other property is relevant in the section.

Reply:
Feel free to open an issue and do that.